frontend/vre/record/record.list.view.js

```diff
@@ -1,5 +1,4 @@
 import Backbone from "backbone";
-import {TabulatorFull as Tabulator} from "tabulator-tables";
```

Comment:
Strictly speaking, you still need this import. You can make Rollup do the right thing by adding `tabulator-tables` to the [external](https://rollupjs.org/configuration-options/#external) and the [output.globals](https://rollupjs.org/configuration-options/#output-globals) in the `rollup.config.js`.